OpenRA.Mods.Common/Scripting/Properties/DemolitionProperties.cs

```diff
@@ -32,7 +32,7 @@ public DemolitionProperties(ScriptContext context, Actor self)
 		public void Demolish(Actor target)
 		{
 			Self.QueueActivity(new Demolish(Self, Target.FromActor(target), info.EnterBehaviour, info.DetonationDelay,
-				info.Flashes, info.FlashesDelay, info.FlashInterval, info.DamageTypes));
+				info.Flashes, info.FlashesDelay, info.FlashInterval, info.DamageTypes, null));
```

Comment:
Why isn't this using the color from the info?

Reply:
Actually I just saw the discussion above. This should probably get a comment so nobody "fixes it" by accident.

Reply:
See discussion above.